### PR TITLE
Home page created

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**
In this PR I have implemented:
Fetch cars
Add buttons
Add social icons
Fetch titles

### CODE REVIEW

**Review by hamidazim321 — CHANGES_REQUESTED** [review]
# 🟦 Request changes
Great work implementing the required design, however, there are some improvements to be made to match the exact design required.

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import FetchCars from '../components/FetchCars';
+
+const HomePage = () => (
+  <div>
+    <FetchCars />
+  </div>
+);
+export default HomePage;
```
Comment (hamidazim321) [inline src/pages/HomePage.jsx:8]: Please add a title for the page to match the design. As you can see on the given design, the page has a title and description. You can use the same title, however, in places where there is "Vespa" in the title, you can change it to "Car" as our app is based on cars.
s

File: src/components/FetchCars.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    currentGroupIndex * groupSize + groupSize
+  );
+
+  useEffect(() => {
+    const fetchCarsData = async () => {
+      const res = await axios.get("https://jsonplaceholder.typicode.com/posts");
+      setCars(res.data);
+    };
+    fetchCarsData();
+  }, []);
```
Comment (hamidazim321) [inline src/components/FetchCars.jsx:42]: ## Optional
As the API endpoints of our backend is completed, you can change the URL to fetch data from our API instead. You can apply this in this PR or in a different PR later, its up to your preference.

Comment (SAZZAD1Q2) [reply src/components/FetchCars.jsx:42]: Thank you for your review.  I appretiate your suggestions. But I could not find backend api ready.  Will tell me on this please?

**Review by hamidazim321 — APPROVED** [review]
# 🥇 APPROVED
Great work. Home page is completed

**Pull request merged**